doc/changelog/04-tactics/13403-occs_nums_nat.rst

```diff
@@ -0,0 +1,8 @@
+- **Removed:**
+  "at @occ_nums" clauses in tactics such as autounfold
+  no longer allow negative values.  A "-" before the
+  list (for set complement) is still supported.  Ex: "at 1 -2"
```

Comment:
This example is bad because it was never supported. `at -1 -2` was but it meant the same thing as `at -1 2`.